### ISSUE TITLE
Consider setting the Current Folder as the default value for the input parameter.

### ISSUE THREAD

**Issue by egvijayanand** [ErikEJ/SqlServer.Rules#132]
At the outset, a nice CLI tool to analyze the SQL files.

Thoroughly deserves a GitHub star.

I've three suggestions:

* Consider setting the **current directory** as the default value for the input parameter similar to how other dotnet CLI commands operate
* File paths in the output log can be **relative to the invocation context**. However, at present, they are displayed as absolute paths, which occupy most of the screen space and make it difficult to follow the actual info
  - Verbose mode can give an absolute path if someone requires that
  - Or set the file paths based on the input value (absolute or relative)
* Include an **info message at the end** to indicate that the analysis is complete if there is no output to display, ensuring that the tool is invoked

**Comment by ErikEJ** [comment]
@egvijayanand Interested in contributing to item 1 and 3 in your list ?? (item 2 requires more consideration)

**Comment by ErikEJ** [comment]
@egvijayanand Actually, item 1 I am not able to replicate (or do not understand). 

I have fixed item 3

I use version 1.0.7

```bash
tsqlanalyze -i Table3.sql

warning: Table3.sql(1,1): Warning SRD0002 : SqlServer.Rules : Table does not have a primary key.
(https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Design/SRD0002.md)
warning: Table3.sql(4,5): Warning SRD0005 : SqlServer.Rules : Avoid the (n)char column type except for
short static length data. (https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Design/SRD0005.md)
warning: Table3.sql(5,5): Warning SRD0005 : SqlServer.Rules : Avoid the (n)char column type except for
short static length data. (https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Design/SRD0005.md)
warning: Table3.sql(1,1): Warning SRP0020 : SqlServer.Rules : Table does not have a clustered index.
(https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Performance/SRP0020.md)
```

**Comment by egvijayanand** [comment]
> @egvijayanand Actually, item 1 I am not able to replicate (or do not understand). 
> 
> I have fixed item 3
> 
> I use version 1.0.7
> 
> ```bash
> tsqlanalyze -i Table3.sql
> 
> warning: Table3.sql(1,1): Warning SRD0002 : SqlServer.Rules : Table does not have a primary key.
> (https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Design/SRD0002.md)
> warning: Table3.sql(4,5): Warning SRD0005 : SqlServer.Rules : Avoid the (n)char column type except for
> short static length data. (https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Design/SRD0005.md)
> warning: Table3.sql(5,5): Warning SRD0005 : SqlServer.Rules : Avoid the (n)char column type except for
> short static length data. (https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Design/SRD0005.md)
> warning: Table3.sql(1,1): Warning SRP0020 : SqlServer.Rules : Table does not have a clustered index.
> (https://github.com/ErikEJ/SqlServer.Rules/blob/master/docs/Performance/SRP0020.md)
> ```

Item 1 refers to invoking `tsqlanalyze` without any parameters. Should pickup all the SQL files in the context and its child directories. For now the `input` parameter is mandatory.

For item 3, invoke with folder as input and not a single file

```shell
tsqlanalyze -i .\
```

Suggested Item 2 for the above scenario.

**Comment by egvijayanand** [comment]
And reg. the info message, if there is no analyzer output for the input then it just returns.

Try with such a file or folder (with `--nologo` this will add much more clarity to this scenario. Typical CI script.)

**Comment by ErikEJ** [comment]
@egvijayanand Thanks, I am fixing the info message
